pkg/manager/manager_test.go

```diff
@@ -1819,7 +1819,7 @@ var _ = Describe("manger.Manager", func() {
 		<-m.Elected()
 
 		Eventually(func() *corev1.Event {
-			evts, err := clientset.CoreV1().Events("").Search(m.GetScheme(), &ns)
+			evts, err := clientset.CoreV1().Events("").SearchWithContext(ctx, m.GetScheme(), &ns)
```

Comment:
fyi. This was not a breaking change in client-go. Search is just deprecated in favor of SearchWithContext